content/learn/07.built-in-libraries/02.i2s/i2s.md

```diff
@@ -29,9 +29,11 @@ The SCK line has a frequency that depends on the sample rate, the number of bits
 
 - **Frequency = SampleRate x BitsPerChannel x numberOfChannels**
 
-In a typical setup, the sender of audio data is called a Transmitter and it transfers data to a Receiver at the other end. The device that controls the bus clock, SCK, together with the Word Select - WS - signal is the Controller in the network and in any network just one device can be Controller at any given time; all the other devices connected are in Peripheral mode. The Controller can be the Transmitter, or the Receiver, or a standalone controller. The digitized audio data sample can have a size ranging from 4 bits up to 32.
+The left and right channels are interleaved in the data stream, and "one sample" is one value for the left channel and one for the right. This means that the FS line oscillates at the sample rate.
```

Comment:
I think this is not 100% needed, could you elaborate this paragraph more maybe?

Reply:
The comment about the FS line is valid only for 2 channels, so scratch that. The rest is exposition about the I2S protocol, and that can also be scrapped, although documentation on the I2S protocol is generally hard to find and difficult to read. This is a tutorial, not a more narrowly focused "howto", and I like to educate people. (Work related habit, do as you like.)